### PR TITLE
DOC: improve multi-version docs

### PR DESCRIPTION
**Pull request by jorgepiloto**
This pull-request adds some improvements to the section on how to enabling multi-version documentation.

### CODE REVIEW

File: doc/source/how-to/documenting.rst [hunk 1]
```diff
@@ -462,75 +462,134 @@ The resulting PDF and intermediate LaTeX files are created in the
...
+              "version_match": get_version_match(__version__),
+          },
+          "navbar_end": ["version-switcher", "theme-switcher", "navbar-icon-links"],
+          ...
+      }
+  
+  
+  .. admonition:: About the ``DCOUMENTATION_CNAME`` environment variable
+  
+      The ``DOCUMENTATION_CNAME`` environmental variable is expected to be
```
Comment (RobPasMue) [inline doc/source/how-to/documenting.rst:501]:
```suggestion
      The ``DOCUMENTATION_CNAME`` environment variable is expected to be
```

File: doc/source/how-to/documenting.rst [hunk 2]
```diff
@@ -462,75 +462,134 @@ The resulting PDF and intermediate LaTeX files are created in the
...
+      declared in the YML file controlling the deployment of the documentation.
+      The idea is that the canonical name (CNAME) is only define in a single
```
Comment (RobPasMue) [inline doc/source/how-to/documenting.rst:503]:
```suggestion
      The idea is that the canonical name (CNAME) is only defined in a single
```

File: doc/source/how-to/documenting.rst [hunk 3]
```diff
@@ -462,75 +462,134 @@ The resulting PDF and intermediate LaTeX files are created in the
...
+          "url": "https://<cname>/dev"
+        }
+      ]
+
+- Enable documentation deployment for development and stable versions, see
+  :ref:`Deploying documentation`.
+
+
+
+With all previous configuration, your project is ready to use multi-version
```
Comment (RobPasMue) [inline doc/source/how-to/documenting.rst:525]:
```suggestion
With all the previous configuration, your project is ready to use multi-version
```

File: doc/source/how-to/documenting.rst [hunk 4]
```diff
@@ -462,75 +462,134 @@ The resulting PDF and intermediate LaTeX files are created in the
...
+documentation in an automated way. This means that, every time you release a
+new version, it will be added to the drop-down button in the documentation page
+of the project.
 
-Add the following workflow job to build and deploy development and stable
-documentation in an automated way.
+.. admonition:: Controlling the desired amount of versions showing up in the drop-down
+
+    Only the development together with the latest three stable versions are
```
Comment (RobPasMue) [inline doc/source/how-to/documenting.rst:532]:
```suggestion
    Only the development branch together with the latest three stable versions are
```

**Review by RobPasMue — APPROVED** [review]
LGTM!

File: doc/source/how-to/documenting.rst (hunk 4 above)
Comment (MaxJPRey) [inline doc/source/how-to/documenting.rst:532]:
```suggestion
    Only the development and the latest three stable versions are
```

File: doc/source/how-to/documenting.rst [hunk 5]
```diff
@@ -462,75 +462,134 @@ The resulting PDF and intermediate LaTeX files are created in the
...
+
+Deploying documentation
+-----------------------
+PyAnsys libraries deploy their documentation online via `GitHub Actions`_ to
+`GitHub Pages`_. This documentation is hosted on the `gh-pages`_ branch of the
+repository of the project. Documentation deployment is done by uploading the
+HTML documentation artifact to the `gh-pages`_ branch of the repository, see
+`enabling GitHub pages`_.
+
+Add the following workflow job to deploy development and stable documentation
```
Comment (MaxJPRey) [inline doc/source/how-to/documenting.rst:557]:
```suggestion
Add the following workflow job to deploy both development and stable documentation
```

**Review by MaxJPRey — APPROVED** [review]
Very helpful.

**Pull request merged**